### PR TITLE
fix: querying all projects don't return system projects

### PR DESCRIPTION
**Pull request by mpro7**
### Task Description/Number

Issue Number: DEV-1943

### Basic Requirements

Please check if your PR fulfills the following requirements:

- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

### PR Type

What kind of change does this PR introduce?

- [x] fix: represents bug fixes
- [ ] refactor: represents production code refactoring
- [ ] feat: represents a new feature
- [ ] docs: documentation changes (no production code change)
- [ ] chore: maintenance tasks (no production code change)
- [ ] test: all about tests: adding, refactoring tests (no production code change)
- [ ] other... Please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No
- [ ] Maybe (not 100% sure => check with FE)

### Does this PR change client-test-data?

- [x] Yes (don't forget to update the JS-LIB team about the change)
- [ ] No

### CODE REVIEW

File: webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala [hunk 1]
```diff
@@ -239,8 +239,10 @@ final case class ProjectsResponderADMLive(
   override def projectsGetRequestADM(): Task[ProjectsGetResponseADM] =
     projectService.findAll
       .flatMap(projects =>
-        if (projects.nonEmpty) { ZIO.succeed(ProjectsGetResponseADM(projects)) }
-        else { ZIO.fail(NotFoundException(s"No projects found")) }
+        if (projects.nonEmpty) {
+          val noSystemProjects: List[ProjectADM] = projects.filter(p => p.id.startsWith("http://rdfh.ch/projects/"))
+          ZIO.succeed(ProjectsGetResponseADM(noSystemProjects))
+        } else ZIO.fail(NotFoundException(s"No projects found"))
```
Comment (BalduinLandolt) [inline webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala:245]: not sure what the intended behaviour is: should `NotFoundException` be returned if there are no projects at all, or if there are system projects but no other projects?

**Review by BalduinLandolt — COMMENTED** [review]
There are still tests failing, so that needs fixing, but so far it looks good

**Review by seakayone — APPROVED** [review]
Please request a review once the tests are fixed.

**Review by seakayone — CHANGES_REQUESTED** [review]
Please request a review once the tests are fixed.

File: webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala (hunk 1 above)
Comment (mpro7) [reply webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsResponderADM.scala]: `NotFoundException` should be returned when there is no projects at all. System projects are supposed to be filtered out from the results anyway, so they shouldn't have been taken into the consideration here.

**Review by BalduinLandolt — APPROVED** [review]
LGTM

**Review by seakayone — APPROVED** [review]

**Pull request merged**